scripts/botlife.coffee

```diff
@@ -17,41 +17,36 @@
 #   macklinu
 
 snackResponses = [
-  "Om nom nom!",
-  "That's very nice of you!",
-  "Oh thx!",
-  "Thank you very much.",
-  "Thanks for the treat!",
+  "Om nom nom!"
+  "That's very nice of you!"
+  "Oh thx!"
+  "Thank you very much."
+  "Thanks for the treat!"
```

Comment:
Commas are [optional](http://coffeescript.org/#literals) in CoffeeScript. Let me double check this against the version of CoffeeScript we are using (but I think it's at the core of the language, not a new feature), or alternately, we could add them back if it's clearer.